### DISCUSSION TITLE
Question regarding parameters in hierarchical model

### DISCUSSION THREAD

**Discussion by SaschaFroelich** [lnccbrown/HSSM#544] [General]
My apologies if this is not HSSM-specific. I have looked at the [bambi-notebooks](https://bambinos.github.io/bambi/notebooks), but so far, I didn't find the answer to this question.

I formulate my model hierarchically for 10 participants, and a factor `jokercondition`, which has 3 levels (A, B, and C). Then for the model parameters I include something like this, in the case of v:

```
include=[
    {
        "name": "v",
        "formula": "v ~ 1 + jokercondition + (0 + jokercondition|participant_id)",
        "prior": {
            "jokercondition|participant_id": {"name": "Normal", 
                                  "mu": {"name": "Normal", "mu": 2., "sigma": 1.},
                                  "sigma": {"name": "HalfNormal", "sigma": .3}},
            "Intercept": {"name": "Normal", "mu": 1, "sigma": 2, "initval": 1},
        },
        "link": "identity",
    }
```

So then there are the following output parameters relating to `v`:

```
v_Intercept
v_jokercondition[B]
v_jokercondition[C]
v_jokercondition|participant_id[A, 1.0]
v_jokercondition|participant_id[A, 2.0]
v_jokercondition|participant_id[A, 3.0]
v_jokercondition|participant_id[A, 4.0]
v_jokercondition|participant_id[A, 5.0]
v_jokercondition|participant_id[A, 6.0]
v_jokercondition|participant_id[A, 7.0]
v_jokercondition|participant_id[A, 8.0]
v_jokercondition|participant_id[A, 9.0]
v_jokercondition|participant_id[A, 10.0]
v_jokercondition|participant_id[B, 1.0]
v_jokercondition|participant_id[B, 2.0]
v_jokercondition|participant_id[B, 3.0]
v_jokercondition|participant_id[B, 4.0]
v_jokercondition|participant_id[B, 5.0]
v_jokercondition|participant_id[B, 6.0]
v_jokercondition|participant_id[B, 7.0]
v_jokercondition|participant_id[B, 8.0]
v_jokercondition|participant_id[B, 9.0]
v_jokercondition|participant_id[B, 10.0]
v_jokercondition|participant_id[C, 1.0]
v_jokercondition|participant_id[C, 2.0]
v_jokercondition|participant_id[C, 3.0]
v_jokercondition|participant_id[C, 4.0]
v_jokercondition|participant_id[C, 5.0]
v_jokercondition|participant_id[C, 6.0]
v_jokercondition|participant_id[C, 7.0]
v_jokercondition|participant_id[C, 8.0]
v_jokercondition|participant_id[C, 9.0]
v_jokercondition|participant_id[C, 10.0]
v_jokercondition|participant_id_mu[A]
v_jokercondition|participant_id_mu[B]
v_jokercondition|participant_id_mu[C]
v_jokercondition|participant_id_sigma[A]
v_jokercondition|participant_id_sigma[B]
v_jokercondition|participant_id_sigma[C]
```

If I interpret this correctly, the level A of the factor `jokercondition` on the subject-level is taken into `v_Intercept`. So is then `v_jokercondition|participant_id_mu[A]` the mean of the group-level distribution of `v_Intercept`? If not, how do these outputs have to be interpreted?

Best,
Sascha

**Comment by frankmj** [comment]
You probably want instead to do

v ~ 1 + jokercondition + (1 + jokercondition|participant_id)


That way you will also have subject specific intercepts (which will be the
values for the reference condition  A).

And then v_intercept will be the mean of the group in that condition, and
the mu’s of the other conditions will be group distributions of within
subject differences relative to that

**Comment by digicosmos86** [comment]
I'll move this general question to discussions so we can use issues to track bugs and feature requests
